trafilatura/core.py

```diff
@@ -488,6 +495,13 @@ def recover_wild_text(tree, result_body, options, potential_tags=TAG_CATALOG):
         search_expr += '|.//div|.//lb|.//list'
     # prune
     search_tree = prune_unwanted_sections(tree, potential_tags, options)
+    # find hljs elements to detect that if it's code
```

Comment:
This seems more like a conversion issue and could be handled elsewhere, what do you think?

Reply:
Yes, I agree. Can you suggest the right place, please? I'm still not highly familiar with the codebase

Reply:
I moved this code part around to use it earlier, i.e. during conversion.